client/go.mod

```diff
@@ -1,10 +0,0 @@
-module github.com/ddelnano/terraform-provider-xenorchestra/client
-
-go 1.20
-
-require (
-	github.com/cenkalti/backoff/v3 v3.2.2
-	github.com/gorilla/websocket v1.4.2
-	github.com/mitchellh/mapstructure v1.5.0
-	github.com/sourcegraph/jsonrpc2 v0.0.0-20210201082850-366fbb520750
-)
```

Comment:
What's the rationale for removing this? This was done to support terraformer (https://github.com/vatesfr/terraform-provider-xenorchestra/pull/138) and there is also a strong likelihood that this code will be shared by the XO packer plugin.

Reply:
well we had import errors when we built the pulumi provider that we could resolve by just having one go.md file. 
I see the need for it now, but maybe moving it into another repository is the way to go then :)

Reply:
Can you share the steps for reproducing the pulumi build issue or share the output of building it against #309? Unless it it isn't possible to make the pulumi work coexist, this must stay for now.

Reply:
@belfhi just checking in on my earlier message. Once that is answered, we can move quickly on merging the last PR.

Reply:
hi @ddelnano so I can confirm that removing the client go.mod is not neccessary. I manages to build the pulumi provider https://github.com/belfhi/pulumi-xenorchestra using a _rewrite_ for the revision.

Reply:
Thanks for confirming :+1: